app/views/home/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>POKe</h1>
+
+<%if !current_user %>
+  <%= render "shared/login" %>
```

Comment:
Refactor to remove !current user (should automatically redirect to posts if signed in)